Content.Server/DoAfter/DoAfterEventArgs.cs

```diff
@@ -105,6 +110,7 @@ public sealed class DoAfterEventArgs
             CancelToken = cancelToken;
             Target = target;
             MovementThreshold = 0.1f;
+            DamageThreshold = 1.0f;
```

Comment:
this should probably default to 0.0, so any amount of damage breaks if BreakOnDamage is set and no threshold is specified

Reply:
also this should be a FixedPoint2 and not float